systemtest/src/test/java/io/strimzi/systemtest/operators/FeatureGatesST.java

```diff
@@ -338,21 +326,15 @@ private void setupClusterOperatorWithFeatureGate(String extraFeatureGates) {
      *                         if the rolling update has been completed across all replicas     .
      */
     private void rollKafkaNodePoolWithActiveProducerConsumer(TestStorage testStorage, int kafkaReplicas) {
-        final String producerName = testStorage.getProducerName() + "-rolling";
-        final String consumerName = testStorage.getConsumerName() + "-rolling";
-
-        // setup clients
-        KafkaClients clients = new KafkaClientsBuilder()
-            .withProducerName(producerName)
-            .withConsumerName(consumerName)
-            .withBootstrapAddress(KafkaResources.plainBootstrapAddress(testStorage.getClusterName()))
-            .withTopicName(testStorage.getTopicName())
-            .withMessageCount(testStorage.getMessageCount())
-            .withDelayMs(500)
-            .withNamespaceName(testStorage.getNamespaceName())
+
+        resourceManager.createResourceWithWait(KafkaTopicTemplates.topic(testStorage.getClusterName(), testStorage.getContinuousTopicName(), 3, 3, 2, testStorage.getNamespaceName()).build());
```

Comment:
I suppose that you are creating KafkaTopic which was created "automatically" when the clients started, right?
Having the constants or variables instead of the magic numbers there would make more sense (in terms of readability)